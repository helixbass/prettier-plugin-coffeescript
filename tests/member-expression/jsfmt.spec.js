run_spec(__dirname, ['coffeescript'], {indentChain: true})
run_spec(__dirname, ['coffeescript'], {indentChain: false})
