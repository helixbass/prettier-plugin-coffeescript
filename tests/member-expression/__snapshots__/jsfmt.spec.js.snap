// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`indent-when-breaking.coffee 1`] = `
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa().bbbbbbbbbbbbbbbbbbbbbbbb
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa()
  .bbbbbbbbbbbbbbbbbbbbbbbb

`;

exports[`indent-when-breaking.coffee 2`] = `
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa().bbbbbbbbbbbbbbbbbbbbbbbb
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa()
.bbbbbbbbbbbbbbbbbbbbbbbb

`;
